reference_guide/api_changes/api_changes_list_2020.md

```diff
@@ -64,6 +64,13 @@ Please see [Incompatible API Changes](/reference_guide/api_changes_list.md) on h
 
 > **NOTE** Changes from API marked with [`org.jetbrains.annotations.ApiStatus.@Experimental/ScheduledForRemoval`](upsource:///platform/util/src/org/jetbrains/annotations/ApiStatus.java) are not listed here, as incompatible changes are to be expected.
 
+# 2020.2
+
+## Changes in IntelliJ Platform 2020.2
+
+`com.intellij.psi.util.PsiTreeUtil.processElements` method parameter type changed from raw `PsiElementProcessor` to `PsiElementProcessor<PsiElement>`
```

Comment:
this needs to match signature
`<class name>.<method name>(<human-readable parameters>) method parameter type changed from <before> to <after>`
exactly

Reply:
Better now?